RoboJackets/robocup-software
# Basic offensive pileup

## Description
Basic Offensive Pileup play. Has one robot capture the ball. One robot goes to a dropback point behind the pileup. One robot goes to the side of the pileup

## Associated Issue
Issue #1348 

## Steps to test
### Test Case 1
1. Run sim
2. Select basic_offensive_pileup
3. run to see if it works as intended.
